services/QuillLMS/app/views/pages/shared/_footer.html.erb

```diff
@@ -1,126 +1,129 @@
-        </li>
-        <li>
-          <a href="/tools/proofreader">Quill Proofreader</a>
-        </li>
-        <li>
-          <a href="/tools/grammar">Quill Grammar</a>
-        </li>
-        <li>
-          <a href="/tools/evidence">Quill Reading for Evidence</a>
-        </li>
-      </ul>
-      <ul>
-        <li>
-          <a href="">EXPLORE CURRICULUM</a>
-        </li>
-        <li>
-          <a href="/activities/packs">Featured Activity Packs</a>
-        </li>
-        <li>
-          <a href="/ap">AP Activities</a>
-        </li>
-        <li>
-          <a href="/preap">Pre-AP Activities</a>
-        </li>
-        <li>
-          <a href="/springboard">SpringBoard Activities</a>
-        </li>
-        <li>
-          <a href="/activities/section/7">ELA Standards</a>
-        </li>
-      </ul>
-      <ul>
-        <li>
-          <a href="">TEACHER CENTER</a>
-        </li>
-        <li>
-          <a href="/teacher-center">All Resources</a>
-        </li>
-        <li>
-          <a href="/teacher-center/topic/getting_started">Getting Started</a>
-        </li>
-        <li>
-          <a href="/teacher-center/topic/teacher_stories">Teacher Stories</a>
-        </li>
-        <li>
-          <a href="/teacher-center/topic/writing_instruction_research">Writing Instruction Research</a>
-        </li>
-        <li>
-          <a href="/faq">FAQ</a>
-        </li>
-        <li>
-          <a href="/premium">Premium</a>
-        </li>
-      </ul>
-      <ul>
-        <li>
-          <a href="">ABOUT US</a>
-        </li>
-        <li>
-          <a href="/about">About Us</a>
-        </li>
-        <li>
-          <a href="/impact">Impact</a>
-        </li>
-        <li>
-          <a href="/pathways">Pathways Initiative</a>
-        </li>
-        <li>
-          <a href="/team">Team</a>
-        </li>
-        <li>
-          <a href="/careers">Careers</a>
-        </li>
-        <li>
-          <a href="/press">Press</a>
-        </li>
-      </ul>
-    </div>
-    <div class="partners-footer">
-      <p>We’re helping every student succeed thanks to our partners:</p>
-      <a href="/about">
-        <div class="partner-links">
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/gates_logo.svg" alt='Gates Foundation Logo'/>
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/google_logo.svg" alt='Google.org Logo'/>
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/footer-att.svg" alt='AT&T Logo'/>
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/footer-fast-forward.svg" alt='Fast Forward Logo'/>
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/footer-nbc.svg" alt='NBC Logo'/>
-          <img class="lazyload" src="https://assets.quill.org/images/homepage/footer-robin-hood.svg" alt='Robinhood Logo'/>
+  <% if current_user&.role == 'student' || current_user&.role == 'staff' %>
```

Comment:
I suspect that this logic is copied from another file, but I'm trying to get us to standardize on `current_user&.student?` and `current_user&.staff?` rather than doing direct comparisons to `role`, so would like to see this updated to `if current_user&.student? || current_user&.staff?`